src/package.ts

```diff
@@ -929,8 +929,10 @@ async function prepublish(cwd: string, manifest: Manifest, useYarn: boolean = fa
 
 	console.log(`Executing prepublish script '${useYarn ? 'yarn' : 'npm'} run vscode:prepublish'...`);
 
-	const { stdout, stderr } = await exec(`${useYarn ? 'yarn' : 'npm'} run vscode:prepublish`, { cwd, maxBuffer: 5000 * 1024 });
+	const { stdout, stderr, error } = await exec(`${useYarn ? 'yarn' : 'npm'} run vscode:prepublish`, { cwd, maxBuffer: 5000 * 1024 });
 	process.stdout.write(stdout);
+	// in case of error, stderr gets written by a top-level exception handler
+	if (error !== undefined) throw error;
 	process.stderr.write(stderr);
```

Comment:
Maybe also print stderr.

```suggestion
	process.stderr.write(stderr);
	// in case of error, stderr gets written by a top-level exception handler
	if (error) {
		throw error;
	}
```

Reply:
Well, I don't know for sure why that happens yet, but (as I said in the comment) stderr gets printed anyway when the exception is thrown and caught somewhere on top. That is, if we print it here explicitly, it gets printed _twice_.

My guess is that the `message` of an `Error` that gets thrown by `exec()` is set to contain the `stderr` of a child process. Or maybe `toString()` of this `Error` gives us that.

Reply:
```
Welcome to Node.js v13.12.0.
Type ".help" for more information.
> const cp = require('child_process')
undefined
> let error
undefined
> void cp.exec('>&2 echo this gets printed to stderr && exit 1', e => error = e)
undefined
> error.toString()
'Error: Command failed: >&2 echo this gets printed to stderr && exit 1\n' +
  'this gets printed to stderr \r\n'
> error.message
'Command failed: >&2 echo this gets printed to stderr && exit 1\n' +
  'this gets printed to stderr \r\n'
```